src/containers/Tenant/Query/QueriesHistory/QueriesHistory.tsx

```diff
@@ -57,6 +59,23 @@ function QueriesHistory({changeUserInput}: QueriesHistoryProps) {
             sortable: false,
             width: 600,
         },
+        {
+            name: 'EndTime',
+            header: i18n('history.endTime'),
+            render: ({row}) =>
+                row.endTime ? formatDateTime(new Date(row.endTime as string).getTime()) : '-',
```

Comment:
`row.endTime.toString()` instead of `row.endTime as string`, it more error proof